templates/frontend/pages/userLogin.tpl

```diff
@@ -37,6 +37,7 @@
 		<input type="hidden" name="source" value="{$source|escape}" />
 
 		<fieldset class="fields">
+			<legend>{translate key="user.userCredentials"}</legend>
```

Comment:
Let's re-use "login" (`{translate key="user.login"}`) as the legend here. I know it's repetitive, but I think it's the most useful description of the fields.